pm_tb_data/fetch/nsidc_0001.py

```diff
@@ -87,7 +87,13 @@ def get_nsidc_0001_tbs_from_disk(
         resolution=resolution,
         hemisphere=hemisphere,
     )
-    ds = xr.open_dataset(filepath, group=sat)
+    try:
+        ds = xr.open_dataset(filepath, group=sat)
+    except OSError as err:
+        raise FileNotFoundError(
```

Comment:
Ideally we'd raise something like a `PmTbDataNotFound` error to handle cases like this consistently. In this case, the file exists, but it doesn't contain the data we're looking for.

OK for now though, since code that relies on `pm_tb_data` currently expects a `FileNotFoundError`.